python/lsst/ts/mtdome/mtdome_csc.py

```diff
@@ -530,6 +540,45 @@ async def write_then_read_reply(
 
             return data
 
+    def assert_moveAz_command_is_same_as_latest(
+        self, position: float, velocity: float
+    ) -> bool:
+        """Assert that the received moveAz command is the same as the latest
+        moveAz command or not.
+
+        Parameters
+        ----------
+        position : `float`
+            The target position to move to.
+        velocity : `float`
+            The velocity at which the target position is moving.
+
+        Returns
+        -------
+        bool
+            True if the issues moveAz command is the same as the latest moveAz
+            command or False otherwise.
+
+        Notes
+        -----
+        The moveAz command is regarded to be the same as the latest one if and
+        only if the position is the same and the velocity is 0.0. In all other
+        cases the command is regarded not to be the same. This is important
+        because, if the velocity != 0.0, the dome is following a moving target
+        and the chance of it being at exactly the commanded position with the
+        commanded velocity can be considered zero and therefore the moveAz
+        command has to be sent to the dome.
+        """
+        if (
+            math.isclose(velocity, 0.0)
+            and math.isclose(position, self.latest_moveAz_command["position"])
+            and math.isclose(velocity, self.latest_moveAz_command["velocity"])
+        ):
```

Comment:
I strongly suggest making abs_tol explicit (I think abs_tol is more meaningful than rel_tol for this particular test) -- either as a pair of module constants (one for position, one for velocity) or a pair of arguments to the method with suitable defaults.

Otherwise there's no obvious way to tune the code to work with whatever limitations the dome has, and readers have to look up the function in the Python docs to see what's going on.

It may suffice to use the same abs_tol for both position and velocity, but by keeping them separate you can more easily tune them and it makes the units clear.

Reply:
Done